apps/desktop/public/theme-init.js

```
@@ -0,0 +1,10 @@
+(function () {
+  var t = 'system';
+  try {
+    t = localStorage.getItem('theme') || 'system';
```

ESLint

[error] 'localStorage' is not defined.

(no-undef)